src/common/databaseFilters.ts

```diff
@@ -8,15 +8,16 @@ export const fullNameSearch = <DbData extends DbClientRow | DbParcelRow>(
     clientIsActiveColumnLabel: Extract<keyof DbData, "is_active" | "client_is_active">
 ): ServerSideFilterMethod<DbData, string> => {
     return (query, state) => {
-        if (state === "") {
+        const newState = state.replace(/[,.():/]/g, "");
```

Comment:
Are these the only special characters we want to remove? Or would we be better off specifying a list of allowed characters?

Reply:
Yea the documentation said these are the only ones that may disrupt the url, but since the filter is for names we might want to only prompt for alphanumeric input / alphabet?

Reply:
Not being the supabase documentation but the PostgREST one:
https://postgrest.org/en/stable/references/api/url_grammar.html#reserved-characters

found in post:
https://stackoverflow.com/questions/76065534/how-to-escape-string-characters-in-supabase-js-or-query